components/organisms-simple/MessageCard/MessageCard.tsx

```diff
@@ -1,14 +1,54 @@
 /** @module MessageCard */
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
-import React, {Fragment} from 'react';
+import React, {FC, Fragment} from 'react';
 import Button from '~components/atoms/Button/Button';
 import '~components/molecules/CardShell/Cards.scss';
 import CardShell from '~components/molecules/CardShell/CardShell';
 import '~components/molecules/CardSummaries/CardSummaries.scss';
 import './MessageCard.scss';
 
-const MessageCard = ({
+type MessageCardProps = {
+  /** Enables browser auto complete. */
+  autoComplete?: boolean;
+  /** Whether it should animate on mount */
+  animate?: boolean;
+  /** To display something after the Submit button. */
+  afterButton?: React.ReactNode;
+  /** To display something before the Submit button. */
+  beforeButton?: React.ReactNode;
+  /** Changes the text in the Submit button. */
+  buttonText?: string;
+  /** Additional class names to apply to the button. */
+  className?: string;
+  /** The cards description. */
+  description?: React.ReactNode | string;
```

Comment:
```suggestion
  description?: React.ReactNode;
```
I don't think you actually need both reactnode and string